src/pyfmi/simulation/assimulo_interface_fmi3.pyx

```diff
@@ -0,0 +1,611 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2025 Modelon AB
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# distutils: define_macros=NPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION
+
+"""
+This file contains code for mapping FMI3 FMUs to the Problem specifications
+required by Assimulo.
+"""
+
+import numpy as np
+cimport numpy as np
+
```

Comment:
FMI2 copy paste; commenting out event, result handling and the likes for now.

Plus removed everything dynamic_diagnostics to log.